process_png_metadata.py

```diff
@@ -97,15 +97,16 @@ def build_file_list(file, tab_index: int, file_list: list[dict]) -> list[dict]:
 
     file = file.name if tab_index == 0 else file
     file_ext = pathlib.Path(file).suffix
+    filename = pathlib.Path(file).name.split(".")[0]
```

Comment:
use stem from pathlib in case of complex file names with multiple "." in them

Reply:
You are right I should have used [0..-1] or it's equivalent.
As you proposed I changed it to .stem instead (I didn't know about it)